src/pages/home/report/ReportActionCompose/SuggestionEmoji.tsx

```diff
@@ -193,6 +194,8 @@ function SuggestionEmoji(
 
     const getSuggestions = useCallback(() => suggestionValues.suggestedEmojis, [suggestionValues]);
 
+    const checkIfSuggestionVisible = useCallback(() => isEmojiSuggestionsMenuVisible, [isEmojiSuggestionsMenuVisible]);
```

Comment:
Our guidelines for booleans are that they should start with `is` or `are` to indicate true/false. I think the following naming makes more sense for what this is doing based on my understanding. In that it will return true/false based on whether the suggestion menu is visible. 

```suggestion
    const isSuggestionsMenuVisible = useCallback(() => isEmojiSuggestionsMenuVisible, [isEmojiSuggestionsMenuVisible]);
```

Reply:
Sure, it makes sense. The only concern is that it's a callback, and it would be quite confusing to do `isSuggestionsMenuVisible()`, so maybe it makes sense to name it `getIsSuggestionsMenuVisible`?

Reply:
Ah cool. That makes sense to me.